.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,71 @@
+name: Behavior Bug
+description: Report issues with behaviour bugs.
+labels: [ "type: bug" ]
```

```suggestion
labels: [ "bug" ]
```
There is no "type: bug" label